node/router/tests/common/mod.rs

```diff
@@ -111,7 +111,7 @@ pub async fn validator(listening_port: u16, max_peers: u16) -> TestRouter<Curren
         sample_account(),
         &[],
         max_peers,
-        false,
+        true,
```

Comment:
Thank you!!

The default for validators is false, so I suggest to add a comment, same as https://github.com/AleoHQ/snarkOS/pull/3163/files#diff-0490634f09b53b897acaf1041eb100043b7bd7627128eb7e77f745df58b64377R62

```suggestion
        true, // This test requires validators to connect to peers.
```